CHANGES

```diff
@@ -3,6 +3,7 @@ znapzend (0.21.3) unstable; urgency=medium
   * Maintenance release: refine splitting of [[user@]host:]dataset[:with-colons][@snap[:with-colons]] strings to work for the realistic majority of use-cases; fix back support of pool root dataset in such spec
   * Update self-tests with verification that [[user@]host:]dataset[:with-colons][@snap[:with-colons]] string decoding yields expected results
   * Fixed CI recipes and contents for spell-checker
+  * Extended handling of "enabled=off" for sub-trees: if the same intermediate dataset declares "recursive=on", the disablement takes place for descendants as well
```

Comment:
maybe explain the idea in a few more words ... also updating the documentation on enabled=off might be helpful for people to understand what is going to happen

Reply:
Good point :)